Wikipedia/Code/MWKSection.m

```diff
@@ -280,7 +280,7 @@ - (NSString*)summary {
 - (nullable NSArray<MWKTitle*>*)disambiguationTitles {
     NSArray* textNodes = [self elementsInTextMatchingXPath:WMFSectionDisambiguationTitlesXPathSelector];
     return [textNodes wmf_mapAndRejectNil:^id (TFHppleElement* node) {
-        if (node.text.length == 0 || [node.text containsString:@"redlink=1"]) {
+        if (node.text.length == 0 || [node.text containsString:@"redlink=1"] || [node.text containsString:@"cite_note"]) {
```

Comment:
sure, but if we had tests, it wouldn't be a "scary thing" that we couldn't change. e.g. the image URL parsing regex was recently changed, and had several regression tests to verify it still covered specific cases